Pandas/movie_lens/u.item

```diff
@@ -540,7 +540,7 @@
 540|Money Train (1995)|01-Jan-1995||http://us.imdb.com/M/title-exact?Money%20Train%20(1995)|0|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0
 541|Mortal Kombat (1995)|01-Jan-1995||http://us.imdb.com/M/title-exact?Mortal%20Kombat%20(1995)|0|1|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0
 542|Pocahontas (1995)|01-Jan-1995||http://us.imdb.com/M/title-exact?Pocahontas%20(1995)|0|0|0|1|1|0|0|0|0|0|0|0|1|0|1|0|0|0|0
-543|Mis�rables, Les (1995)|01-Jan-1995||http://us.imdb.com/M/title-exact?Mis%E9rables%2C%20Les%20%281995%29|0|0|0|0|0|0|0|0|1|0|0|0|1|0|0|0|0|0|0
+543|Mis�rables, Les (1995)|01-Jan-1995||http://us.imdb.com/M/title-exact?Mis%E9rables%2C%20Les%20%281995%29|0|0|0|0|0|0|0|0|1|0|0|0|1|0|0|0|0|0|0
```

Comment:
You shouldnt be touching this file. It seems your system can't handle unicode, hence the "??".

Revert all changes to this file.